cli.sh

```diff
@@ -1821,6 +1821,14 @@ git hooks config [enable|disable|print] fail-on-non-existing-shared-hooks [--glo
 Enable or disable failing hooks with an error when any
 shared hooks configured in \`.shared\` are missing,
 which usually means \`git hooks update\` has not been called yet.
+
+git hooks config [yes|no|reset|print] delete-detected-lfs-hooks
+
+By default, detected LFS hooks during install are disabled and backuped.
```

Comment:
Can you change `backuped` to `backed up` (and `backing up`) all throughout the PR please?

Reply:
Oh, god, yes :)

Reply:
Do you want to replace the remaining `backuped` instances with `backed up` ? :)
Otherwise this looks good, I can merge it afterwards.